projects/guava/UrlEscapersFuzzer.java

```diff
@@ -0,0 +1,81 @@
+import com.code_intelligence.jazzer.api.FuzzedDataProvider;
+import com.code_intelligence.jazzer.api.FuzzerSecurityIssueMedium;
+import com.code_intelligence.jazzer.api.FuzzerSecurityIssueLow;
+import com.google.common.base.Charsets;
+import com.google.common.escape.Escaper;
+import com.google.common.net.PercentEscaper;
+import com.google.common.net.UrlEscapers;
+import java.lang.IllegalArgumentException;
+import java.net.URLDecoder;
+
+public class UrlEscapersFuzzer {
+
+	/*
+	 * these constants are private members copy-pasted from
+	 * com.google.common.net.UrlEscapers
+	 */
+	static final String URL_FORM_PARAMETER_OTHER_SAFE_CHARS = "-_.*";
+	static final String URL_PATH_OTHER_SAFE_CHARS_LACKING_PLUS =
+      "-._~" // Unreserved characters.
```

Comment:
Unsure if this is proper indentation, could you please check? Thanks.

Reply:
Reintended the file